src/modules/blocks/ticket/container-header/price/style.pcss

```diff
@@ -38,3 +39,16 @@
 		}
 	}
 }
+
+.tribe-editor__ticket__container-header-price-available {
+	color: #727272;
+	display: block;
+	font-family: var(--tec-font-family-sans-serif);
+	font-size: var(--tec-font-size-1);
+	font-style: normal;
+	font-weight: var(--tec-font-weight-regular);
+	line-height: 17px;
+	line-height: 14px;
```

Comment:
Which is it?

Reply:
Also see comment above on line heights 😉

Reply:
Thanks, this was fixed